vignettes/ParametricCopulaFamilies.Rmd

```diff
@@ -0,0 +1,163 @@
+---
+title: "Parametric Copula Families and Their Functions"
+params:
+  do_calc: FALSE
+output:
+  bookdown::html_vignette2:
+    toc: true
+author: "Alan Kuchinsky, Elif F. Acar, Martin Lysy"
+date: 2024-03-26
+bibliography: copulaReferences.bib
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Parametric Copula Families and Their Functions}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+header-includes:
+  - \setmathfont{STIXTwoMath-Regular.otf}
+  - \usepackage{xcolor}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, echo = FALSE}
+library(LocalCop)
+```
+
+
+This document will discuss the implementations of the copula families seen in this package. That is, we will discuss the CDF, conditional CDF, also known as the h-functions in uniform scale, and PDF of each of the families implemented. For the conditional CDF, most of the copula implemented are symmetric in $u$ and $v$, so it is sufficient to provide one conditional CDF, and a simple swap of the variables will provide the other conditional CDF. Where this is not the case, both conditional CDF's will be provided. The document will offer both the original-scale and the logarithm-scale version of each formula where applicable. It should also be noted that the implementation utilizes the logarithm-scale version where possible to increase accuracy.
+
+## Gaussian Copula
+
+The Gaussian copula family has a single parameter, which is the correlation between the two normal variables, $\rho \in (-1, 1)$. That is, the correlation matrix is:
+
+$\Sigma =\begin{bmatrix}
+1 & \rho\\
+\rho & 1\\
+\end{bmatrix}$
+
+With determinant $1 - \rho^2$. It should also be noted that the Gaussian CDF has no closed-form CDF, as it is built around normal CDF's. However, using $\Phi_2$ as the bivariate normal CDF with the correlation matrix above and $\Phi^{-1}$ as the quantile function of the standard univariate normal distribution, we can write the CDF as follows:
+
+$$C(u, v; \rho) = \Phi_2 ( \Phi^{-1}(u), \Phi^{-1}(v) ; \rho)$$
```

Comment:
For display-style equations (on their own line), it's customary to put the environment delimiters (`$$` in this case) on separate lines to make the math easier to read.  Also, equations are always considered part of a sentence, and therefore should contain punctuation (should end with a "." in this case).  Also, this means more often than not you don't use a ":" to introduce the equation (think of how rarely you use colons in regular writing).